downstream/assemblies/eda/assembly-eda-set-up-rhaap-credential.adoc

```diff
@@ -2,6 +2,14 @@
 
 = {PlatformName} credential
 
-The {PlatformName} credential type can connect to {ControllerName} through the use of an {ControllerName} URL and a username and password. After you have created this credential type, it can be attached to a rulebook and used to run rulebook activations.  
+When {EDAcontroller} is deployed on {PlatformNameShort} {PlatformVers}, you can create a {PlatformName} credential to connect to {ControllerName} through the use of an {ControllerName} URL and a username and password. After it has been created, you can attach the {PlatformName} credential to a rulebook and use it to run rulebook activations. These credentials provide a simple way to configure communication between {ControllerName} and {EDAcontroller}, enabling your rulebooks to launch jobs.
```

Comment:
Small nit, and I may be incorrect here, but wondering if we should instead say - 

```suggestion
When {EDAcontroller} is deployed on {PlatformNameShort} {PlatformVers}, you can create a {PlatformName} credential to connect to {ControllerName} through the use of an {ControllerName} URL and a username and password. After it has been created, you can attach the {PlatformName} credential to a rulebook and use it to run rulebook activations. These credentials provide a simple way to configure communication between {ControllerName} and {EDAcontroller}, enabling your rulebook activations to launch job templates.
```

Reply:
Thanks for the correction here, Zack. I want to be sure to correctly state this for the user's perspective.